cms/static/js/views/course_info_update.js

```diff
@@ -17,32 +23,53 @@ define(['js/views/validation', 'codemirror', 'js/models/course_update',
 
             initialize: function() {
                 this.template = this.loadTemplate('course_info_update');
-                this.render();
```

Comment:
Yes.  It was being called twice, which was causing resets when it was not needed.  

@andy-armstrong @cahrens Why is there a need to call render in the init with Backbone?  I tested around this and the page seems to still function as expected.